util-images/containerd/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM docker:latest
```

Comment:
I think we agreed that we don't need to base it on the docker img, right?

Reply:
ah, i tried alpine but i couldn't get ctr running on it. any suggestions there?

Reply:
What about golang-alpine? I believe ctr is written in golang, isn't it?

Reply:
i tried golang:1.12.1-alpine, same issue there. its rather odd, let me dig into it a bit more

Reply:
because i believe the docker base image was also based off alpine

Reply:
i can make it work with ubuntu:xenial, its even smaller in size than the previous alpine, golang-alpine, docker images

Reply:
What is the exact error you're getting for golang-alpine?

Reply:
i get ctr command not found. so if you `ls -l` in the `/bin` folder of an alpine container, all the utilities are symlinked to `/bin/busybox`, i suspect it has something to do with that.. because all my non-busybox utils do not work

Reply:
I see. I guess there must be some missing runtime dependency in the golang-alpine image. The regular golang seems to be working: 
```
FROM golang:1.13.5

RUN wget https://github.com/containerd/containerd/releases/download/v1.4.3/containerd-1.4.3-linux-amd64.tar.gz
RUN tar xvf containerd-1.4.3-linux-amd64.tar.gz
ENTRYPOINT bin/ctr
```

What's the difference in size of the image when using ubuntu and golang?

Reply:
im getting 
ubuntu 299mb
golang 934mb